### ISSUE TITLE
[Suggestion] Standardise action names for "moving the cursor"

### ISSUE THREAD

**Issue by miblodelcarpio** [arybczak/ncmpcpp#182]
It occurs to me that we have three different words merely in #181 for moving the "cursor": `move_end`, `scroll_up_album` and `jump_to_playing_song`. May it be worth standardising these to `scroll_*`? I can appreciate the `jump_to_browser`, `jump_to_media_library`, etc. actions having different names, to highlight the notion that they "jump" between screens, but I think having different names for variants on essentially the same action within the same screen, and the `move_*` actions clashing with `move_selected_items_to`, only has the potential to cause confusion.

**Comment by arybczak** [comment]
I agree about `move_home` and `move_end`. As for jump_to_playing_song, it only scrolls in playlist, it might change directories/other columns in browser/media library.
